sass/layout/_layout.scss

```diff
@@ -0,0 +1,112 @@
+.container {
+	display: grid;
+	height: calc(100vh - 2rem);
+	grid-gap: 0.5rem;
+	grid-template-columns: 16rem auto;
+	grid-template-rows: 5rem auto 5rem;
+}
+
+.header__logo-box {
+	grid-column: 1 / 2;
+}
+
+.header__navbar {
+	grid-column: 2 / 3;
+}
+
+.content {
+	grid-column: 2 / 3;
+	padding: 2rem;
+	display: grid;
+	grid-gap: 1rem;
+	grid-template-columns: calc(28% - 0.7rem) calc(40% - 0.7rem) calc(32% - 0.7rem);
+	grid-template-rows: auto auto;
+
+	.content__item-1 {
+		grid-column: 1 / 2;
+	}
+
+	.content__item-2 {
+		grid-column: 2 / 3;
+	}
+
+	.content__item-3 {
+		grid-column: 1 / 3;
+	}
+
+	.content__item-4 {
+		grid-column: 3 / 4;
+		grid-row: 1 / 3;
+	}
+}
+
+.footer {
+	grid-column: 1 / 3;
+}
+
+@media only screen and (max-width: 575px) {
+	.content {
+		grid-template-columns: 1fr;
+		grid-template-rows: auto auto auto auto;
+		padding: 0.1rem !important;
```

Comment:
Avoid using !important if it is not 100% necessary.
Here you can first of all specify the parent class (as you have parent selector in the next media query which overwrites current code). And second, switch your media queries places. The smallest screen styles should be the last as you set up just the maximum size limit

Reply:
This one. I've spent so much time figuring out why the next media query was overwriting the padding. I just didn't notice that I had the ".container" parent selector in it. Thank you. Also switched the media queries order.

Reply:
It happens))